xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamPVRManager.cpp

```diff
@@ -243,23 +243,6 @@ CDVDInputStream::ENextStream CDVDInputStreamPVRManager::NextStream()
   return NEXTSTREAM_NONE;
 }
 
-bool CDVDInputStreamPVRManager::CanRecord()
-{
-  if (!m_isRecording)
```

Comment:
This member can be removed, right?

Reply:
nope, m_isRecoding indicates if a recordings is playing and is still in use

Reply:
okay, then